package.json

```diff
@@ -17,55 +17,58 @@
     "jshint-watch": "gulp jshint-watch"
   },
   "dependencies": {
-    "bows": "1.4.8",
+    "bows": "1.6.0",
     "crossfilter": "1.3.12",
-    "d3": "3.5.9",
-    "d3.chart": "0.2.1",
-    "duration-js": "3.3.4",
+    "d3": "3.5.17",
+    "d3.chart": "0.3.0",
+    "duration-js": "3.6.0",
     "lodash": "3.10.1",
-    "moment-timezone": "0.4.1",
+    "moment-timezone": "0.5.4",
     "sundial": "1.5.0"
   },
   "devDependencies": {
     "autoprefixer-loader": "3.2.0",
-    "babel-core": "5.8.34",
-    "babel-loader": "5.3.3",
-    "babel-runtime": "5.8.34",
+    "babel-core": "6.11.4",
+    "babel-loader": "6.2.4",
+    "babel-plugin-rewire": "1.0.0-rc-4",
```

Comment:
Yeah, I just ran through `next-update` and found a lot of dependencies that could be updated without breaking or modifying any tests. I'll drop the list in a larger comment and you can take a glance to see if things would be worth updating. This one stood out to me first since it's an `rc` status.